src/toniarts/openkeeper/view/PlayerCameraState.java

```diff
@@ -212,6 +212,7 @@ public void doTransition(int sweepFileId, final ActionPoint point) {
             @Override
             public void onPlay(CinematicEvent cinematic) {
                 stateManager.getState(PlayerState.class).setTransitionEnd(false);
+                stateManager.getState(PlayerState.class).setWideScreen(true);

```

Comment:
As far as i can tell this is the case in the original game as well. For example the goblin intro in level1, there is also the cinematic screen but there is no trigger for setting it to widescreen.